.ruff.toml

```diff
@@ -316,3 +316,6 @@ unfixable = [
 "astropy/wcs/*" = ["I001", "C408"]
 "docs/*" = []
 "examples/coordinates/*" = []
+
+# Never try to fix these.
+"docs/_static/*" = []
```

Comment:
@nstarman or @eerovaher , this didn't work. How do I ask pre-commit.ci to ignore files in `docs/_static`?

Reply:
This won't work because we'd have to add every rule to this list. The idea above to add to pre-commit's exclude list is the right approach.

Reply:
@nstarman , I don't speak the language of ruff or pre-commit, can you pls tell me exactly what I should do here? Thanks!